package pt.ulusofona.cm.kotlin.challenge.exceptions

class VeiculoLigadoException : Exception("Não é possível ligar este veiculo, pois o mesmo já se encontra ligado.")